test/e2e/Apps/BasicDotNetIsolated/HelloCities.cs

```diff
@@ -54,5 +54,27 @@ public static async Task<HttpResponseData> HttpStart(
             // See https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-http-api#start-orchestration
             return await client.CreateCheckStatusResponseAsync(req, instanceId);
         }
+
+        [Function("HelloCities_HttpStart_Scheduled")]
+        public static async Task<HttpResponseData> HttpStartScheduled(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+            [DurableClient] DurableTaskClient client,
+            FunctionContext executionContext,
+            DateTime ScheduledStartTime)
```

Comment:
nit: method parameters should be camelCase: `scheduledStartTime`.

Reply:
Done